homeassistant/components/deconz/deconz_event.py

```diff
@@ -44,7 +50,7 @@ def async_add_sensor(sensors, new=True):
 async def async_unload_events(gateway) -> None:
     """Unload all deCONZ events."""
     for event in gateway.events:
-        event.async_will_remove_from_hass()
+        await event.async_will_remove_from_hass()
```

Comment:
Do we need to do this sequentially or can we await all events concurrently with `asyncio.gather`?

Reply:
No connection between them so I can change it

Reply:
Fixed in #40640, there was no need to keep it async so "fixed?!"